errors.go

```diff
@@ -11,9 +12,9 @@ func readFailed(w http.ResponseWriter, err error) {
 	http.Error(w, "cannot read request body", http.StatusInternalServerError)
 }
 
-func bookNotFound(w http.ResponseWriter) {
-	log.Event(nil, "book not found in list", log.INFO)
-	http.Error(w, "book not found", http.StatusNotFound)
+func bookNotFound(w http.ResponseWriter, id string) {
+	log.Event(nil, fmt.Sprintf("book with id=%s not found in list", id), log.INFO)
+	http.Error(w, fmt.Sprintf("book id %q not found", id), http.StatusNotFound)
```

Comment:
Functionally fine but you could use reuse the message in both the log and http error. Why do more work than you have to? 😁 
```
msg := fmt.Sprintf("book id %q not found", id)
log.Event(nil, msg, log.INFO)
http.Error(w, msg, http.StatusNotFound)
```

Reply:
🤯 so DRY. Thank you 🙏